core/src/main/java/com/google/instrumentation/stats/StatsManager.java

```diff
@@ -28,4 +28,9 @@
    * Returns the current stats data, {@link View}, associated with the given {@link ViewDescriptor}.
    */
   public abstract View getView(ViewDescriptor viewDescriptor);
+
+  /**
+   * Returns the default {@link StatsContextFactory}.
+   */
+  public abstract StatsContextFactory getStatsContextFactory();
```

Comment:
`StatsContextFactory` is already exposed and is used by gRPC.  I agree that it probably needs to be changed, though, so I'll save this change for another PR.